package.json

```diff
@@ -40,7 +40,18 @@
   "jest": {
     "preset": "ts-jest",
     "projects": [
-      "<rootDir>/packages/*"
+      "<rootDir>/packages/append-collection",
```

Comment:
Do we need this change please? It's a bit annoying if we need to maintain this list explicitly as we add packages rather than having it work for all packages by default.

Reply:
Unfortunately, I couldn't find any other way to prevent jest from running the mocha tests and causing failures. I'll test a little bit more today to make sure however.